### PR TITLE
Bump GHW to version 0.11.0

### PR DESCRIPTION
**Pull request by rbaturov**
Some security issues have been reported by SAST that are caused due to unsanitized input occurring in ghw/pkg/block/block_linux.go file.
For more info check out this Jira ticket:
https://issues.redhat.com/browse/OCPBUGS-25137

I believe this issue will resolved, as that problematic input line is no longer there in the proposed version.

### CODE REVIEW

**Review by ffromani — COMMENTED** [review]
please explicitely mention jira tickets (besides the reference in the PR title, a necessary evil for automation) only if they contain publicly accessible content

File: go.mod [hunk 1]
```diff
@@ -7,8 +7,9 @@ require (
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
 	github.com/coreos/ignition v0.35.0
 	github.com/coreos/ignition/v2 v2.15.0
+	github.com/go-logr/stdr v1.2.2
 	github.com/google/go-cmp v0.5.9
-	github.com/jaypipes/ghw v0.8.1-0.20210605191321-eb162add542b
+	github.com/jaypipes/ghw v0.11.0
```
Comment (ffromani) [inline go.mod:12]: I don't see how a bump should fix the referenced issue. Let's discuss offline.

**Comment by rbaturov** [comment]
Closing this PR, as to support the newer version, more files need to be collected by must gather.

**Pull request closed without merging**